cmd/syncer/main.go

```diff
@@ -28,8 +28,9 @@ const (
 )
 
 var (
-	kubeconfig = flag.String("kubeconfig", "", "Config file for -from cluster")
-	clusterID  = flag.String("cluster", "", "ID of this cluster")
+	kubeconfig   = flag.String("from", "", "Kubeconfig file for -from cluster")
+	toKubeconfig = flag.String("to", "", "Kubeconfig file for -to cluster. If not set, the InCluster configuration will be used")
```

Comment:
What do you think about `-from_kubeconfig` and `-to_kubeconfig` as flag names? `-from` and `-to` on their own seem a bit too vague.

Reply:
I agree, especially because we might want to also add `from_Context` and `to_Context` to override the context that we want to refer inside the KubeConfig file, which would be useful to specify in which logical cluster we want to watch on the KCP side.

I'll do that.

Reply:
Done